src/utils/keystore.ts

```diff
@@ -186,13 +253,17 @@ export async function recover (password, keyObject) {
  * @return {Object}
  */
 export async function dump (
-  name,
-  password,
-  privateKey,
-  nonce = nacl.randomBytes(24),
-  salt = nacl.randomBytes(16),
-  options = {}
-) {
+  name: string,
+  password: string,
+  privateKey: string | Buffer,
+  nonce: Buffer | Uint8Array = nacl.randomBytes(24),
+  salt: Buffer | Uint8Array = nacl.randomBytes(16),
+  options: {
+    kdf?: string,
+    cipher?: string,
+    kdfparams?: KdfParams
```

Comment:
```suggestion
    kdf_params?: KdfParams
```
This is used in deriveKey which accepts it as kdf_params - let's follow snake_case convention for it